src/main/thrift/proxy.thrift

```diff
@@ -346,7 +337,6 @@ service AccumuloProxy {
   )
 
   void addSplits (
-    1:binary login
```

Comment:
I think you're probably going to need to keep the `login` field, but use it differently. It is currently used to track authenticated sessions. I think we still need that, so the preferred language client can authenticate to the proxy with some kind of shared secret. The task at hand is to eliminate the need for the preferred language client to instruct the proxy on how to authenticate to Accumulo, because the proxy should be able to authenticate to Accumulo using the client properties file. But, we still want the proxy to not just allow anybody to connect to it and use it. So, the preferred language client should have some way to authenticate to the proxy. For that, this thrift field can be reused to pass that secret shared between the preferred language client and the proxy.